pledgeindia/pledge/models.py

```diff
@@ -0,0 +1,30 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from user.models import User
+from base.models import TimeStampedModel
+
+
+# Create your models here.
+class Pledge(TimeStampedModel):
+    """Pledge Model"""
+
+    title = models.CharField(_('title'), max_length=100)
+    description = models.TextField(_('description'), null=True)
+
+    class Meta:
+        verbose_name = _('pledge')
+        verbose_name_plural = _('pledges')
+
+    def __str__(self):
+        return self.title
+
+
+class UserPledges(TimeStampedModel):
+    """User Pledges Model"""
+
+    pledge = models.ForeignKey(Pledge, on_delete=models.CASCADE)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    datetime_taken = models.DateTimeField(_('taken on'), auto_now_add=True)
```

Comment:
`taken_on`/`taken_at` as a field name sounds better here. 
In the help text section, `taken_on` doesnt really signify anything. Update the fields help text to `datetime at which user took the pledge.`